### PR TITLE
feat: add decisions from 2022-02-07 general assembly

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

**Review by azjezz — COMMENTED** [review]
should we start splitting this file per date? e.g: `profile/scop/decisions/2022-02-07.md`? makes it easier to determine when a decision was voted on.

File: profile/scop/decisions.md [hunk 1]
```diff
@@ -2,22 +2,41 @@
 
 Cette page regroupe les décisions importantes déjà discutées en AG :
 
+## Vie de la coopérative
+
+### Assemblées Générales
+
+- Les coopérateur⋅rices doivent se réunir dans les bureaux de leur antenne respective pour suivre l'AG à distance.
```
Comment (K-mos) [inline profile/scop/decisions.md:9]: de préférence

File: profile/scop/decisions.md [hunk 2]
```diff
@@ -2,22 +2,41 @@
...
+- Une fois par an, une AG est organisée dans une antenne autre que Lille.
```
Comment (K-mos) [inline profile/scop/decisions.md:10]: pour chaque antenne

File: profile/scop/decisions.md [hunk 3]
```diff
@@ -2,22 +2,41 @@
...
+
+- Chaque coopérateur⋅rice reçoit une indemnité télétravail de 20 € par mois.
+- Il est possible de se faire rembourser (ou de demander) du matériel utilisé pour le télétravail (écran, chaise, bureau, etc.).
+- Il est possible de demander une prime forfait mobilités durables (limite de 500 € par an).
 
 ### Bonus
 
 - Chèques-cadeaux de 173 € environ pour Noël.
 - Chèques culture de 100 €.
+- Tickets restaurant de 7,50 € (en chèque ou en carte).
```
Comment (K-mos) [inline profile/scop/decisions.md:39]: 9€

File: profile/scop/decisions.md [hunk 4]
```diff
@@ -40,3 +59,7 @@ Cette page regroupe les décisions importantes déjà discutées en AG :
 - 10000 € pour financer la fondation PHP.
 - 10000 € pour financer la fondation CNCF.
 - Enveloppe globale de 5000 € par an pour financer des logiciels libres.
+
+## Parts
+
+- 2000 € de parts dans le café participatif de Bois Blancs (SCIC).
```
Comment (K-mos) [inline profile/scop/decisions.md:65]: peut-être préciser que ce sont des parts détenues par la Scop ? et dans ce cas ajouter notre SCI ?

**Review by K-mos — COMMENTED** [review]
merci pour les ajouts !

**Comment by alanpoulain** [comment]
> should we start splitting this file per date? e.g: `profile/scop/decisions/2022-02-07.md`? makes it easier to determine when a decision was voted on.

The aim is to see at a glance all the important past decisions. To see when a decision was made, we can use git blame or search in the reports.

**Pull request merged**